src/Components/RightSidebar/RequestSignatureTab.vue

```diff
@@ -58,6 +58,11 @@
 				@click="validationFile()">
 				{{ t('libresign', 'Validate') }}
 			</NcButton>
+			<NcButton v-if="filesStore.canValidate()"
+				variant="primary"
+				@click="validationFile()">
```

Comment:
Will be necessary implement a method to this and don't will be the method validationFile(). By example, could be the openFile method.
```suggestion
				@click="openFile()">
```